model_analyzer/analyzer.py

```diff
@@ -64,7 +70,8 @@ def __init__(self, config, server, state_manager, checkpoint_required):
         self._result_manager = ResultManager(config=config,
                                              state_manager=self._state_manager)
 
-    def profile(self, client, gpus):
+    def profile(self, client: Union[TritonGRPCClient, TritonHTTPClient],
+                gpus: list, mode: str, verbose: bool):
```

Comment:
Type hint also supports `List` that allows us to specify the object types inside the list as well.

Reply:
Updated to use List